witness/golang/omniwitness/distributor_configs/witness.yaml

```diff
@@ -8,13 +8,11 @@ Logs:
   - Origin: rekor.sigstore.dev - 3904496407287907110
     PublicKeyType: ecdsa
     PublicKey: rekor.sigstore.dev+c0d23d6a+AjBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABNhtmPtrWm3U1eQXBogSMdGvXwBcK5AW5i0hrZLOC96l+smGNM7nwZ4QvFK/4sueRoVj//QP22Ni4Qt9DPfkWLc=
-    URL: https://rekor.sigstore.dev
-    ID: 3904496407287907110
+    URL: https://rekor.sigstore.dev?treeID=3904496407287907110
```

Comment:
Would it be better to add another configuration field rather than overload URL?

Reply:
My thinking was that the ID is (at least logically) part of the URL - if the log were exposed as a REST API it'd be something like `rekor.example.com/log/<treeID>/get-root` etc., just here it's being sent as a GET param...